geomesa-core/src/main/scala/org/locationtech/geomesa/core/iterators/TemporalDensityIterator.scala

```diff
@@ -30,125 +28,62 @@ import org.apache.commons.codec.binary.Base64
 import org.geotools.feature.simple.SimpleFeatureBuilder
 import org.geotools.geometry.jts.JTSFactoryFinder
 import org.joda.time.{DateTime, Interval}
-import org.locationtech.geomesa.core._
-import org.locationtech.geomesa.core.index.{IndexEntryDecoder, _}
-import org.locationtech.geomesa.feature._
-import org.locationtech.geomesa.utils.geotools.{SimpleFeatureTypes, TimeSnap}
-import org.opengis.feature.simple.SimpleFeatureType
+import org.locationtech.geomesa.core.index.getDtgFieldName
+import org.locationtech.geomesa.core.iterators.FeatureAggregatingIterator.Result
+import org.locationtech.geomesa.core.iterators.TemporalDensityIterator.TimeSeries
+import org.locationtech.geomesa.utils.geotools.TimeSnap
 
-import scala.util.Random
+import scala.collection.mutable
```

Comment:
i don't think so, this is just to make it so that you can refer to mutable.HashMap instead of collection.mutable.HashMap